docs/pydoc/config/generators_api.yml

```diff
@@ -0,0 +1,30 @@
+loaders:
+  - type: haystack_pydoc_tools.loaders.CustomPythonLoader
+    search_path: [../../../]
+    modules:
+      [
+        "haystack_experimental.components.generators.chat.openai",
+      ]
+    ignore_when_discovered: ["__init__"]
+processors:
+  - type: filter
+    expression:
+    documented_only: true
+    do_not_filter_modules: false
+    skip_empty_modules: true
+  - type: smart
+  - type: crossref
+renderer:
+  type: haystack_pydoc_tools.renderers.ReadmeCoreRenderer
+  excerpt: Enables text generation using LLMs.
+  category_slug: haystack-api
+  title: Generators
+  slug: generators-api
+  order: 20
+  markdown:
+    descriptive_class_title: false
+    classdef_code_block: false
+    descriptive_module_title: true
+    add_method_class_prefix: true
+    add_member_class_prefix: false
+    filename: generators_api.md
```

Comment:
Just for my own knowledge. I've seen other pydoc configs add `experimental_` to the beginning of filename. Is that not necessary?

Reply:
You are right. Fixed in https://github.com/deepset-ai/haystack-experimental/pull/292/commits/d45a03b111c0f001869b14b1706a4faf70759775.